api/app/signals/apps/questionnaires/tests/test_models.py

```diff
@@ -157,6 +158,21 @@ def test_get_reachable_from_question_graph(self):
         questions = Question.objects.get_reachable_from_question_graph(question_graph)
         self.assertEqual(len(questions), 1)
 
+    def test_draw_base64(self):
+        question_graph = QuestionGraph.objects.get(name='one')
```

Comment:
Aside: not problem in this PR but worth mentioning:

I will probably change these tests a little, the `create_<something>` functions should return a graph instance. That instance can then be saved on the TestCase and accessed directly through `self.graph_<something>`. We can then skip the `QuestionGraph.objects.get(name='<something>')` step.

Reply:
We can fix this in a PR adding this behaviour for the tests. Perhaps make some Factories that represent a certain shape and questionnaire.